.circleci/src/workflows/scheduled_native_promotion.yml

```diff
@@ -18,12 +18,12 @@ jobs:
       context:
         - mattermost
         - tuerantuer-google-play
-  - promote_android:
-      name: promote_aschaffenburg_android
-      build_config_name: aschaffenburg
-      context:
-        - mattermost
-        - tuerantuer-google-play
+#  - promote_android: # TODO IGAPP 567 Uncomment
```

Comment:
IGAPP-567 is the current task, I'm a bit confused :thinking:

Reply:
:D we have to disable it first before we can enable it again... I think "Arbeitsbeschaffungsmaßnahme" is the right term for this.